frontend/package.json

```diff
@@ -152,5 +152,9 @@
     "ignorePatterns": [
       "cypress"
     ]
+  },
```

Comment:
For frontend we where using only `devDependencies`. I'm all for separating build tools from bundled dependencies, maybe in a separate PR or with tech debt task.

Reply:
Moved